chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,67 @@
+"""DNS A record update.
+
+This file has the function to update the DNS A record
+
+Typical usage example in an experiment json file:
+
+  "module": "chaosgcp.dns.actions",
+  "func": "update_service",
```

Comment:
It's fantastic you added a docstring header!

However, the module docstring isn't about listing what the file contains (this should be exposed in the `__all__` attribute). Instead, the top level docstring, when present, should explain the goal of of the module in the context of chaos engineering. Why would you need to change the dns? what's the purpose of this?